### DISCUSSION TITLE
how to calculate addition with watch and function

### DISCUSSION THREAD

**Discussion by fadil9872** [react-hook-form/react-hook-form#8128] [Q&A] [answered]
I have a lot of data for example calculate 1 and calculate2 how do I add up these 2 numbers

I'm tired and confused why I made unlimited loops

link : https://codesandbox.io/s/react-hook-form-focus-forked-d8wvt3?file=/src/index.js

**Comment by bluebill1049** [comment, marked as answer]
I don't think you need watch at all, you can do that onChange with each inputs.

**Reply by fadil9872** [reply]
can you look at my code again i have changed it, why i use watch with the same name
